app/swagger/schemas/vet360/address.rb

```diff
@@ -6,300 +6,103 @@ module Vet360
+          block.property :address_line3, type: :string, maxLength: 100
+          block.property :address_pou,
+                         type: :string,
+                         enum: ::Vet360::Models::Address::ADDRESS_POUS,
+                         example: ::Vet360::Models::Address::RESIDENCE
+          block.property :address_type,
+                         type: :string,
+                         enum: ::Vet360::Models::Address::ADDRESS_TYPES,
+                         example: address_type
+          block.property :city, type: :string, example: 'Fulton', maxLength: 100
+          block.property :country_code_iso3,
+                         type: :string,
+                         example: 'USA',
+                         minLength: 3,
+                         maxLength: 3
         end
+        # rubocop:enable Metrics/MethodLength
 
-        swagger_schema :PostVet360DomesticAddress do
-          key :required, %i[
-            address_line1
-            address_pou
-            address_type
-            city
-            country_name
-            state_code
-            zip_code
-          ]
-          property :address_line1,
-                   type: :string,
-                   example: '1493 Martin Luther King Rd',
-                   maxLength: 100
-          property :address_line2, type: :string, maxLength: 100
-          property :address_line3, type: :string, maxLength: 100
-          property :address_pou,
-                   type: :string,
-                   enum: ::Vet360::Models::Address::ADDRESS_POUS,
-                   example: ::Vet360::Models::Address::RESIDENCE
-          property :address_type,
-                   type: :string,
-                   enum: ::Vet360::Models::Address::ADDRESS_TYPES,
-                   example: ::Vet360::Models::Address::DOMESTIC
-          property :city, type: :string, example: 'Fulton', maxLength: 100
-          property :country_name,
-                   type: :string,
-                   example: 'United States',
-                   pattern: ::Vet360::Models::Address::VALID_ALPHA_REGEX.inspect
-          property :state_code,
-                   type: :string,
-                   example: 'MS',
-                   minLength: 2,
-                   maxLength: 2,
-                   pattern: ::Vet360::Models::Address::VALID_ALPHA_REGEX.inspect
-          property :zip_code,
-                   type: :string,
-                   example: '38843',
-                   maxLength: 5,
-                   pattern: ::Vet360::Models::Address::VALID_NUMERIC_REGEX.inspect
+        def self.domestic_address_fields(block)
+          block.property :state_code,
+                         type: :string,
+                         example: 'MS',
+                         minLength: 2,
+                         maxLength: 2,
+                         pattern: SwaggerHelpers.convert_regex(::Vet360::Models::Address::VALID_ALPHA_REGEX)
```

Comment:
This seems like it must change the schema -- how is the new pattern different from the old one? Same question for all three instances.

Reply:
@johnpaulashenfelter 
the old pattern was incorrect, when running specs against the old pattern it would throw this error `'//v0/profile/address_validation#/addresses/0/address/zip_code' value "11249" did not match the regex '/[0-9]+/'`

according to swagger docs the pattern needs to be in javascript format and not include the starting and ending `/` characters https://swagger.io/docs/specification/data-models/data-types/

the reason the specs don't fail in master is because the `Vet360AddressSuggestion` schema doesn't include the patterns, once you add a pattern in with the old format then the specs fail